spec/context_spec.rb

```diff
@@ -157,14 +157,20 @@
     expect(context.outcome).to eq(::LightService::Outcomes::SUCCESS)
   end
 
+  it "can contain false values" do
+    context = LightService::Context.make(:foo => false)
+    expect(context[:foo]).to eq false
+  end
+
   context "when aliases are included via .make" do
     let(:context) do
       LightService::Context.make(
         :foo => "foobar",
+        :foo2 => false,
```

Comment:
I believe it should be `:foo2`.  The test ensures that you can retrieve a false value via the alias, which has always worked.  (Looking up a false value via the key failed because the second half of the `||` would execute and overwrite the retrieved value.)

I can remove the test if you don't think it's helpful.